### PR TITLE
Implement ToGlib and FromGlib traits to cmp::Ordering

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/translate.rs [hunk 1]
```diff
@@ -213,6 +214,19 @@ impl ToGlib for Option<char> {
...
 
+impl ToGlib for Ordering {
+    type GlibType = i32;
+
+    #[inline]
+    fn to_glib(&self) -> i32 {
+        match *self {
+            Ordering::Less => -1,
+            Ordering::Equal => 0,
+            _ => 1,
```
Comment (gkoz) [inline src/translate.rs]: Um, could you spell out the last variant please? They deserve equal treatment ;)

Comment (GuillaumeGomez) [inline src/translate.rs]: I'll update the commit's comment as well to try to get redemption haha.

File: src/translate.rs [hunk 2]
```diff
@@ -644,6 +658,19 @@ impl FromGlib<u32> for char {
...
+    #[inline]
+    fn from_glib(val: i32) -> Ordering {
+        if val < 0 {
+            Ordering::Less
+        } else if val > 0 {
+            Ordering::Greater
+        } else {
+            Ordering::Equal
+        }
+    }
```
Comment (gkoz) [inline src/translate.rs]: Want a clever hack? ;) You can also say `val.partial_cmp(&0)`

Comment (GuillaumeGomez) [inline src/translate.rs]: Nice to know but I prefer to keep the current code. :) (it's slower)

Comment (gkoz) [inline src/translate.rs]: Actually now that I've thought a bit more it's not really a hack: `<` and `>` are sugar for `partial_cmp` anyway :)

Comment (GuillaumeGomez) [inline src/translate.rs]: Yes. ^^'

Comment (gkoz) [inline src/translate.rs]: I forgot `unwrap` though, `partial_cmp` returns `Option`. It wouldn't be that pretty after all.

Comment (GuillaumeGomez) [inline src/translate.rs]: Ah! :D

**Pull request merged**